src/ray/raylet/node_manager.cc

```diff
@@ -1209,6 +1310,8 @@ void NodeManager::FinishAssignedTask(Worker &worker) {
     // If this was an actor creation task, then convert the worker to an actor.
     auto actor_id = task.GetTaskSpecification().ActorCreationId();
     worker.AssignActorId(actor_id);
+    auto driver_id = task.GetTaskSpecification().DriverId();
+    worker.AssignDriverId(driver_id);
```

Comment:
this is not needed because we don't unassign driver id for actors, right?

Reply:
yes.